assets/thanos-ruler/thanos-ruler.yaml

```diff
@@ -64,14 +69,19 @@ spec:
     certFile: /etc/tls/grpc/server.crt
     keyFile: /etc/tls/grpc/server.key
   image: quay.io/openshift/origin-thanos:latest
+  queryConfig:
+    key: query.yaml
+    name: thanos-ruler-query-config
   queryEndpoints:
-  - dnssrv+_http._tcp.thanos-querier.openshift-monitoring.svc.cluster.local
+  - ""
```

Comment:
In `query.yaml` I see:

```
  "static_configs":
  - "thanos-querier.openshift-monitoring.svc:9091"
```

I believe this is quite different from the above which implies DNS discovery. What is the difference?

Reply:
Ah yes this was the bug fix I did in prom-operator, the queryEndpoints could not be left empty so this is the workaround. But I believe this fix should be in our payload already.

Reply:
Done.